src/plugins/intel_gpu/src/graph/impls/onednn/gemm_onednn.cpp

```diff
@@ -130,6 +148,19 @@ struct gemm_onednn : typed_primitive_onednn_impl<gemm> {
         }
     }
 
+    static dnnl::memory::desc get_input_memory_desc(const dnnl::memory::dims& dims,
+                                                    dnnl::memory::data_type dt,
+                                                    dnnl::memory::format_tag fmt,
+                                                    const dnnl::memory::dims& strides) {
+        dnnl::memory::desc res;
+        if (strides.empty()) {
+            res = dnnl::memory::desc(dims, dt, fmt);
+        } else {
+            res = dnnl::memory::desc(dims, dt, strides);
```

Comment:
Um is it okay with stride which is not applying the formats set for fused permutes? Could you please check/

Reply:
Ummmm I guess that, currently permute fused inputs are not with padding, to still it is okay..right?